src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7074,6 +7074,12 @@ public partial class ObjectHandle : System.MarshalByRefObject
 }
 namespace System.Runtime.Serialization
 {
+    public sealed partial class DeserializationBlockedException : System.Exception
```

Comment:
Serializable attribute here?

Reply:
Reference assemblies do not have Serializable attributes.

Reply:
Ah yes